.github/sanity-tests

```diff
@@ -0,0 +1,250 @@
+s3tests_boto3/functional/test_s3.py::test_100_continue
```

Comment:
Maybe it would be better to mark them in the test code with the @pytest.mark.sanity decorator ?

Reply:
Yeah, that'd be much easier to maintain.

Reply:
Fixed. Removed this test list and marked tests with @attr('sanity') in a separate commit

Reply:
https://github.com/nspcc-dev/s3-tests/pull/36